itests/fevm_test.go

```diff
@@ -0,0 +1,170 @@
+package itests
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"encoding/hex"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/lotus/itests/kit"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/filecoin-project/go-state-types/big"
+	builtintypes "github.com/filecoin-project/go-state-types/builtin"
+	"github.com/filecoin-project/go-state-types/builtin/v10/eam"
+	"github.com/filecoin-project/lotus/chain/actors"
+	"github.com/filecoin-project/lotus/chain/types"
+	cbg "github.com/whyrusleeping/cbor-gen"
+)
+
+// TestFEVMBasic does a basic fevm contract installation and invocation
+func TestFEVMBasic(t *testing.T) {
+	_ = os.Setenv("BELLMAN_NO_GPU", "1")
```

Comment:
Why do we need to do this?

Reply:
I dont know, its from other tests to not try to use gpu for crypto. Maybe it is not necessary.

Reply:
I removed it, doesnt seem to be doing anything anyway. Let's get rid of all this cargo cult stuff :)